src/codeu/chat/client/commandline/Chat.java

```diff
@@ -370,6 +370,20 @@ public void invoke(Scanner args) {
         }
       }
     });
+    
+    panel.register(“info”, new Panel.Command() {
+  @Override
+  public void invoke(Scanner args) {
+    final ServerInfo info = context.getInfo();
+    if (info == null) {
+      // Communicate error to user - the server did not send us a valid
+              // info object.
+    } else {
+      // Print the server info to the user in a pretty way
```

Comment:
You actually need implement this like the comments say. Look at the version check PR:

https://github.com/lianasyrkett/CodeU-Summer-2017/pull/1/files#diff-085ee4c5309974341970845d11bcb286R101